src/popup/router/components/RadioButton.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <label
+    class="radio-container"
+    :class="{ disabled }"
+  >
+    <input
+      :disabled="disabled"
+      :value="value"
+      :type="type"
+      :name="name"
+      @change="handleChange"
+    >
+    <span
+      class="radio-dot"
+      :class="{ checked: value }"
+    />
+    <slot class="radio-holder" />
+  </label>
+</template>
+
+<script>
+export default {
+  props: {
+    value: { type: [String, Number, Boolean], default: '' },
+    type: { type: String, default: 'checkbox' },
```

Comment:
Input type radio has only one type: `radio`. I think this prop makes no sense.

Reply:
I see that it's used in few places as a checkbox or as a radio (same with checkbox that has different styles) so I think id should stay like that